app/components/filters/candidate_attributes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Filters
+  class CandidateAttributes < ViewComponent::Base
+    attr_accessor :filters
+
+    def initialize(filters:)
+      @filters = filters
+      super
```

Comment:
```suggestion
			super
      @filters = filters
```
Always call super **before** initialising instance variables